src/ICeWall/doc/set_bump_options.md

```diff
@@ -0,0 +1,60 @@
+## set_bump_options
+
+### Synopsis
+```
+  % set_bump_options \
+        [-pitch pitch] \
+        [-spacing_to_edge spacing] \
+        [-bump_pin_name pin_name] \
+        [-cell_name bump_cell_table] \
+        [-num_pads_per_tile value] \
+        [-rdl_layer name] \
+        [-rdl_width value] \
+        [-rdl_spacing value] \
```

Comment:
This should probably check and give warning if values are above the max width or below min width and spacing. Currently does not.

Reply:
Check added